src/components/question/Question.styles.ts

```diff
@@ -78,6 +78,12 @@ export const styles = {
     justifyContent: 'space-between',
     alignItems: 'center'
   },
+  singleAnswer: {
+    padding: '9px 0px 9px 0px'
```

Comment:
```suggestion
    padding: theme.spacing(1, 0)
```

Use spacings from our theme, please. I think you can round 9 pixels to 8.